npc/quests/quests_moscovia.txt

```diff
@@ -7524,6 +7592,8 @@ moscovia,178,127,0	script	The Blacksmith#rus06	1_M_SMITH,{
 			mes "And don't forget to wear these!";
 			delitem Steel,25;
 			rhea_rus_main = 8;
+			if (BaseClass != Job_Thief)
```

Comment:
soo should i keep it or remove it :3 this commit is just for adding quest log to help players locating where they are in the quest i have noticed few other thinks that need to be updated in this script but that for other time.